src/program.js

```diff
@@ -287,6 +287,10 @@ Example: $0 --help run.
           describe: 'Watch for file changes and re-build as needed',
           type: 'boolean',
         },
+        'overwrite-dest': {
```

Comment:
Can you add `alias: 'o'` to this? This will be a commonly used option so I think it needs an alias.